lib/rucio/rse/protocols/protocol.py

```diff
@@ -492,6 +492,14 @@ def delete(self, path):
         """
         raise NotImplementedError
 
+    def bulk_delete(self, pfns):
```

Comment:
I am not sure if we need to add this to the protocol interface, since it is really something more special to globus.